### DISCUSSION TITLE
@Predictor in  @IsotropicStrainHardeningMisesCreep ?

### DISCUSSION THREAD

**Discussion by GhiathMonnet** [thelfer/tfel#539] [General]
Dear All, 
I am using @DSL IsotropicStrainHardeningMisesCreep and have a simple questions.
In the internal Newton algoyithm, I suppose that the first estimation of dp is dp = 0 at the begening of the algorythm. Would it be possible to force mfronts to start from anoter estimation ? I mean is there equivalent commande of @Predictor in @DSL Implicit ?  
Cheers.
Ghiath

**Comment by thelfer** [comment]
@GhiathMonnet Unfortunately not. 

The algorithm search for the increment of the equivalent plastics strain. The initial guess is hard coded to:

~~~~
(seq_e / (3 * mu * theta)) / 2;
~~~~

where `seq_e` is the elastic prediction of the von Mises stress.
